### ISSUE TITLE
empty trash does not empty trash

### ISSUE THREAD

**Issue by LargePrime** [jpfleury/corbeille-spacefm#4]
Have an attached usb drive, and it will not empty trash.  Main drive does delete when requested, and files can be manually deleted

**Comment by jpfleury** [comment]
Yes, it's by design. Trashing files from a removable device is tricky and conflicts with several different expectations from users. See the section [Trashable files](https://github.com/jpfleury/corbeille-spacefm/blob/master/README.md#trashable-files) in the README for a full reasoning. However, I'm open to a pull request that would handle this case ergonomically.
